core/ui_events.js

```diff
@@ -48,6 +48,12 @@ Blockly.Events.Ui = function(opt_block, opt_element, opt_oldValue,
 };
 Blockly.utils.object.inherits(Blockly.Events.Ui, Blockly.Events.Abstract);
 
+/**
+ * Whether or not the event is a UI event.
+ * @type {boolean}
+ */
+Blockly.Events.Ui.prototype.IS_UI_EVENT = true;
```

Comment:
Why is this a constant?

Reply:
I made it a constant on the prototype because its the same for every instance of the class.